src/analysis/seminorm.lean

```diff
@@ -312,6 +320,66 @@ nonneg_of_mul_nonneg_left h zero_lt_two
 
 lemma sub_rev : p (x - y) = p (y - x) := by rw [←neg_sub, p.neg]
 
+variables {α : Type*} [semilattice_sup α] {a b c d : α}
+variables (h₁ : a = b)
+
+#check le_antisymm_iff.mp h₁
+
+#check sup_le_sup
+theorem sup_eq_sup (h₁ : a = b) (h₂ : c = d) : a ⊔ c = b ⊔ d :=
```

Comment:
This lemma doesn't say anything interesting, `rw [h₁, h₂]` closes the goal, which doesn't use any fact about `sup`. So you can replace the place where you use this with just two rewrites.